bin/.gitignore

```diff
@@ -0,0 +1,2 @@
+/main/
+/test/
```

Comment:
I think /bin should be in the root .gitignore file instead

Reply:
Just added bin folder and removed bin/.gitignore.